src/OrleansLoggerProvider/OrleansLoggingFactoryExtension.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Logging;
+using Orleans.Runtime;
+
+namespace Orleans.Extensions.Logging
+{
+    public static class OrleansLoggingFactoryExtension
```

Comment:
This is an ```ILoggingBuilder``` extension class and not ```ILoggingFactory```, please add ```ILoggingFactory``` implementation. SeriLog can be a good sample here perhaps: https://github.com/serilog/serilog-extensions-logging/tree/dev/src/Serilog.Extensions.Logging

Reply:
see my comment above